src/test/java/Search.java

```diff
@@ -0,0 +1,59 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() {
+        //GIVEN
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        //WHEN
+        searching.searchBoxMany();
+
+        //THEN
+        searching.resultFromSearch();
+    }
+    @Test(enabled = true,priority=1)
+    public void isSongSearchwithSentiveCase() {
+        //GIVEN
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        //WHEN
+
+        searching.searchBoxforCaseSensitive();
+
+        //THEN
+        searching.songResultpage();
+        Assert.assertFalse(searching.negativeSearchResult());
+    }
+
+     @Test(enabled= true, priority= 2)
+     public void clearQuery() {
+        //GIVEN
+         LoginPageSearch Login = new LoginPageSearch(driver);
+         SearchPageSearch searching = new SearchPageSearch(driver);
+         Login.login();
+         searching.searchBox("Dark Days");
+
+         //WHEN
+         WebElement InputField = driver.findElement(By.cssSelector("input[name='q']"));
+         InputField.sendKeys((Keys.chord(Keys.COMMAND, "a", Keys.BACK_SPACE)));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/211280785-0d76efa2-7814-4cd6-a60d-bcbd277c2459.png)

Reply:
I moved that part to SearchPageSearch under method clearInputQuery.